makefile

```diff
@@ -119,25 +120,30 @@ TARGET_SRC += ${TARGET_PATH}/PeripheralPins.c
 TARGET_SRC += ${TARGET_PATH}/system_clock.c
 
 include ${MBED_PATH}/mbedfile
+include ${HW_BRIDGE_PATH}/hwbridgefile
 
 UWRT_C_OBJ := $(subst ../,${OBJ_PATH}/,$(UWRT_SRC_C:.c=.o))
 UWRT_CPP_OBJECTS := $(subst ../,${OBJ_PATH}/,$(UWRT_SRC_CPP:.cpp=.o))
 TARGET_OBJ := $(subst ../,${OBJ_PATH}/,$(TARGET_SRC:.c=.o))
 MBED_OBJ := $(subst ../,${OBJ_PATH}/,$(MBED_OBJ))
+HW_BRIDGE_OBJ := $(subst ../,${OBJ_PATH}/,$(HW_BRIDGE_OBJ))
 
 OBJECTS += ${UWRT_C_OBJ}
 OBJECTS += ${UWRT_CPP_OBJECTS}
 OBJECTS += ${TARGET_OBJ}
 OBJECTS += ${MBED_OBJ}
+OBJECTS += ${HW_BRIDGE_OBJ}
 
 
 INCLUDE_PATHS += -I$(CONFIG_PATH)
 INCLUDE_PATHS += -I$(LIB_PATH)
 INCLUDE_PATHS += -isystem$(MBED_PATH)
+INCLUDE_PATHS += -isystem$(HW_BRIDGE_PATH)
+#DO I NEED THIS?
 INCLUDE_PATHS += -isystem$(MBED_INC)
 INCLUDE_PATHS += -isystem${TARGET_PATH}
 INCLUDE_PATHS += $(APP_INC) $(LIB_INC)
-
+#DO I NEED THIS?
```

Comment:
yes you do. the linker script is what tells the linker to place the code in the right memory regions